exercises/secret-handshake/src/example/kotlin/Signal.kt

```diff
@@ -0,0 +1,5 @@
+enum class Signal {
```

Comment:
Oh, grr, it looks like I just did a soft link before for this same thing:

https://github.com/exercism/kotlin/blob/02ecc5a70719f0b0e741992f33879f880f7c407f/exercises/allergies/src/example/kotlin/Allergen.kt